insty-frontend/src/app/_components/social/SocailLogin.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from "next/image";
+import React from "react";
+
+function SocailLogin() {
```

Comment:
P1. 소셜 영문 오타난 것 같아용! 

```suggestion
function SocialLogin() {
```

Reply:
헉 바로 수정하겠습니다.